apps/src/templates/instructions/InlineAudio.jsx

```diff
@@ -210,6 +243,49 @@ class InlineAudio extends React.Component {
     });
   }
 
+  // adds event listeners to the DOM which trigger audio
+  // when a significant enough user interaction has happened
+  addAudioAutoplayTrigger() {
+    AUDIO_ENABLING_DOM_EVENTS.forEach(event => {
+      this.autoplayTriggerElement.addEventListener(event, this.autoplayAudio);
+    });
+  }
+
+  removeAudioAutoplayTrigger() {
+    AUDIO_ENABLING_DOM_EVENTS.forEach(event => {
+      this.autoplayTriggerElement.removeEventListener(
+        event,
+        this.autoplayAudio
+      );
+    });
+  }
+
+  playAudio() {
```

Comment:
What happens if playAudio() is called more than once?  (I think there are some potential race conditions where this might happen?)